README.md

```diff
@@ -90,7 +89,83 @@ configure ESLint for Angular projects** especially when migrating from TSLint. [
     ng g @angular-eslint/schematics:convert-tslint-to-eslint <PROJECT NAME>
     ```
 
-## Recommended Development Environment Configuration
+## Usage
+
+After following the above steps to install and configure the linter, you be able to run it from the command line using `npx eslint .`
+
+### NPM commands
+
+To avoid developers needing to remember tooling-specific commands, each project should add standard aliases to its `package.json`:
+
+```json
+{
+  "scripts": {
+    "lint": "eslint .",
+    "lint-fix": "eslint . --fix",
+  }
+}
+```
+
+This allows developers to lint using `npm run lint` and to run the automatic fixer using `npm run lint-fix`.
+
+### Add linting to your PR build
+
+Each project's pull request build pipeline should ensure no lint errors can be committed to the repository. Invoke `npm run lint` from your GitHub Actions or Azure Pipelines YML.
+
+## Tips for Adopting this Style Guide
+
+### Enable as early as possible
+
+**New projects** should turn on linting before writing any code. It's easier to fix violations as developers add new code than it is to fix large numbers of lint errors across an existing codebase. 
+
+**Existing projects** are likely to have numerous violations even if they already used a different linter (for example, the deprecated [TSLint](https://www.npmjs.com/package/tslint)) as this ruleset is more strict than most. The recommended flow for adopting this ruleset in an existing repository is:
+1. Install the tooling as described above.
```

Comment:
Why are all the list numbers 1. here and below?

Reply:
As we discussed in person, it's a markdown feature where they still render as 2, 3, 4 but it allows you to re-order them more easily when editing.

But I agree there are good reasons to spell it out so I made that change.